product_set/models/product_set.py

```diff
@@ -32,14 +32,20 @@ class ProductSet(models.Model):
         "it's going to be available for all of them.",
     )
 
-    def name_get(self):
-        return [(rec.id, rec._name_get()) for rec in self]
+    display_name = fields.Char(
+        compute="_compute_display_name",
+        string="Display Name",
+        store=False,
+        translate=True,
```

Comment:
These 2 attributes are useless.

Reply:
perfect... was not sure about the necessity. Now i know it!
Thank you!